config/components/prometheus/monitor.yaml

```diff
@@ -17,4 +14,5 @@ spec:
         insecureSkipVerify: true
   selector:
     matchLabels:
-      control-plane: controller-manager
```

Comment:
IIUC we can remove the `labels.contro-plane` label thanks to the use of commenLabels in the `kustomization.yaml`, but if so, why the labels are added here explicitly?

Reply:
You're right, these labels should not theoretically be added explicitly. Unfortunately, Kustomize common labels plugin does not currently process label selectors in custom resources like ServiceMonitor.

Reply:
sgtm, thanks for explaining